### ISSUE TITLE
Highlight last opened file

### ISSUE THREAD

**Issue by mesonium** [FossifyOrg/Gallery#351]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Gallery/issues) or [closed](https://github.com/FossifyOrg/Gallery/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Gallery/discussions) or [closed](https://github.com/FossifyOrg/Gallery/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Gallery/blob/master/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

Highlight the file (e.g. with an outline) when opening it.

### Why do you want this feature?

It's not uncommon to collect very many pictures and videos in the gallery over time, which may also may look similar if taken at the same spot or time for instance.

When you then open a file, e.g. a video in an external player or a picture in the app itself, and then go back to the raster or list view, you may be a bit lost to find exactly the one you just opened, to continue with the next.

If the last opened file were highlighted though, it would be directly clear.

### Additional information

On the desktop, e. g. Dolphin on KDE Plasma, supports this and it's super useful.

**Comment by ismailghedamsi** [comment]
Yeah, please implement this. It's a basic and important features that no gallery app implemented as far as i know.
We could get inspired by how it is implemented in this comic app

![Image](https://github.com/user-attachments/assets/d5fc799a-f8c2-42a8-b274-ed21859730af)
